extensions/datacube/README.md

```diff
@@ -36,40 +41,46 @@ A spatial dimension in vertical (z) direction.
-| reference_system | string\|number   | The spatial reference system for the data, specified as [EPSG code](http://www.epsg-registry.org/) or [PROJ definition](https://proj4.org/operations/projections/index.html). Defaults to EPSG code 4326. |
+| reference_system | string\|number\|object | The spatial reference system for the data, specified as [EPSG code](http://www.epsg-registry.org/), [WKT2 (ISO 19162) string](http://docs.opengeospatial.org/is/18-010r7/18-010r7.html), [PROJJSON](https://proj.org/projjson.html) or [PROJ definition](https://proj.org/usage/quickstart.html). Defaults to EPSG code 4326. |
 
-An Vertical Spatial Dimension Object MUST specify an `extent` or a set of `values`. It MAY specify both.
+An Vertical Spatial Dimension Object MUST specify an `extent` or a set of `values`. It MAY specify both. 
 
 ### Temporal Dimension Object
 
-A temporal dimension based on the ISO 8601 standard. The temporal reference system for the data is expected to be ISO 8601 compliant (Gregorian calendar / UTC). Data not compliant with ISO 8601 can be represented as an *Additional Dimension Object* with `type` set to `temporal`.
+A temporal dimension based on the ISO 8601 standard. The temporal reference system for the data is expected to be ISO 8601 compliant (Gregorian calendar / UTC). Data not compliant with ISO 8601 can be represented as an *Additional Dimension Object* with `type` set to `temporal`.
 
 | Field Name | Type           | Description                                                  |
 | ---------- | -------------- | ------------------------------------------------------------ |
 | type       | string         | **REQUIRED.** Type of the dimension, always `temporal`.      |
+| description | string        | Detailed multi-line description to explain the dimension. [CommonMark 0.29](http://commonmark.org/) syntax MAY be used for rich text representation. |
 | extent     | [string\|null] | **REQUIRED.** Extent (lower and upper bounds) of the dimension as two-dimensional array. The dates and/or times must be strings compliant to [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601). `null` is allowed for open date ranges. |
 | values     | [string]       | If the dimension consists of set of specific values they can be listed here. The dates and/or times must be strings compliant to [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601). |
 | step       | string\|null   | The space between the temporal instances as [ISO 8601 duration](https://en.wikipedia.org/wiki/ISO_8601#Durations), e.g. `P1D`. Use `null` for irregularly spaced steps. |
 
 ### Additional Dimension Object
 
-An additional dimension that is not `spatial`.
+An additional dimension that is not `spatial`, but may be `temporal` if the data is not compliant with ISO 8601.
```

Comment:
Should this be 'is compliant' instead of 'is not compliant?

Reply:
That `not` is there by intention. The Temporal Dimension Object allows ISO8601/Gregorian and if the data is not compliant with that, you need to use the Additional Dimension Object.